python/ModelTools.py

```
@@ -1034,7 +1034,7 @@ def doCombination(self):
     def doModelConfigs(self):
         if not self.options.bin:
             raise RuntimeException
```

Ruff (0.14.2)

`RuntimeException` may be undefined, or defined from star imports

(F405)